inputs/LoginForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { Text, View, TextInput, TouchableOpacity } from 'react-native';
```

Comment:
Duplicate file, needs removal

Reply:
Deleted duplicate file in most recent commit